README.md

```diff
@@ -86,6 +162,36 @@ trainer.register_validation_hook(validation_dataset)
 trainer.train(train_dataset)
 ```
 
-## padertorch.Configurable
+See the [trainer](padertorch/train/trainer.py#L40) for an explanation of its signature.
+If you want to use `pt.io.get_new_storage_dir` to manage your experiments, you have to define an environment variable `STORAGE_ROOT` which points to the path where all your experiments will be stored, i.e., in the example above, a new directory under `$STORAGE_ROOT/my_experiment_1` will be created.
+Otherwise, you can use `pt.io.get_new_subdir` where you can directly input the path to store your model without defining an environment variable.
+
+# Features for Application in Deep Learning
+
+Padertorch provides a selection of frequently used network architectures and functionalities such as activation and normalization, ready for you to integrate into your own models.
+
+- [Multi-layer Feed-Forward](padertorch/modules/fully_connected.py): Multiple fully-connected layers with non-linearity and dropout.
+- [CNN](padertorch/contrib/je/modules/conv.py) (currently subject to breaking changes and hardly any documentation): 1d- and 2d-CNNs with residual connections, dropout, gated activations, batch and sequence norm and correct handling of down- and upsampling.
+- [Normalization](padertorch/modules/normalization.py): Perform normalization of arbitrary axes/dimensions of your features, keep track of running statistics and apply learnable affine transformation.
+- [Collection of activation functions](padertorch/ops/mappings.py): Fast access of various activation functions with just a string.
+- [Losses](padertorch/ops/losses): We provide an implementation of the [Kullback-Leibler divergence](padertorch/ops/losses/kl_divergence.py) and different [regression](padertorch/ops/losses/regression.py) objectives.
```

Comment:
Should we add examples of the regression losses? Something like "and different regressions objectives (SI-SNR, log-MSE and others)". Many people coming here for source separation probably want to use SI-SNR and won't know that we classified them as regression losses.